File: Web/RMA-Web-Shell/projects/shared-components-lib/src/lib/rules-engine/shared/models/last-viewed-item.ts
import { ItemType } from './item-type.enum';

export class LastViewedItem {
    id: number;
    itemId: number;
    itemType: ItemType;
    user: string;
    date: Date;
    code: string;
    name: string;
    description: string;
    startDate: Date;
    endDate: Date;
    canEdit: boolean;
}

File: Service/Admin/RMA.Service.Admin.RulesManager.Contracts/Entities/LastViewedItem.cs
using RMA.Service.Admin.RulesManager.Contracts.Enums;

using System;

namespace RMA.Service.Admin.RulesManager.Contracts.Entities
{
    public class LastViewedItem
    {
        public int Id { get; set; }
        public int ItemId { get; set; }
        public RulesItemType ItemType { get; set; }
        public string User { get; set; }
        public DateTime Date { get; set; }
        public string Code { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public bool CanEdit { get; set; }
    }
}